res/locales/tr/modules.go

```diff
@@ -0,0 +1,252 @@
+package tr
```

Comment:
Either you remove modules which contains additional translations, or you remove them.
Please read the comments of each module.

Reply:
I fixed.